Dockerfile.py3.deploy

```diff
@@ -0,0 +1,104 @@
+FROM python:3.6-slim-stretch
```

Comment:
Can you make the naming consistent with the one we use for local dev, which is `Dockerfile.python3` ? Obviously the CircleCI config would then need to be updated.


 (FWIW the reason it wasn't called `Dockerfile.py3` was because some editors consider `.py3` files as python files by default)